lib/reek/ast/sexp_extensions/constant_assignment.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+module Reek
+  module AST
+    module SexpExtensions
+      # Utility methods for :casgn nodes.
+      module CasgnNode
+        def send_expression
```

Comment:
This expression is defined as `#value` in `lib/reek/ast/sexp_extensions/module.rb`